src/metabase/driver/sql/query_processor.clj

```diff
@@ -107,7 +107,7 @@
 
 (defmulti ->integer
   "Cast to integer"
-  {:arglists '([driver value])}
+  {:arglists '([driver honeysql-expr])}
```

Comment:
I changed some of these to make it clearer that they expect Honey SQL rather than MBQL